assets/js/index.js

```diff
@@ -6,74 +6,86 @@ const addBtn = document.getElementById('add-btn');
+      bookList.children[i].id = i;
+    });
 
-// Create a Book
-function createBook(id, title, author) {
-  const bookElement = `
-    <div id="${id}" class="book">
-        <p class="book-title">${title}</p>
-        <p class="book-author">${author}</p>
-        <button class="remove-btn" type="button">Remove</button>
-    </div>`;
+    localStorage.setItem('Books', JSON.stringify(booksArr));
+  }
 
-  bookList.insertAdjacentHTML('beforeend', bookElement);
+  // create a Book
+  create(id = this.id, title = this.title, author = this.author) {
+    const bookElement = `
+    <div id="${id}" class="book">
+        <div class="book-details">
+          <p class="book-title">${title}</p>
+          <span class="by">&nbsp;by&nbsp;</span>
+          <p class="book-author">${author}</p>
+        </div>
+        <button class="btn remove-btn" type="button">Remove</button>
+    </div`;
 
-  const removeBtn = document.getElementsByClassName('remove-btn');
+    bookList.insertAdjacentHTML('beforeend', bookElement);
 
-  removeBtn[removeBtn.length - 1].addEventListener('click', removeBook);
-}
+    const removeBtn = document.getElementsByClassName('remove-btn');
 
-// Load Books
-function loadBooks(bookData = JSON.parse(localStorage.getItem('Books'))) {
-  if (bookData !== null) {
-    bookData.forEach((book) => {
-      createBook(book.id, book.title, book.author);
-    });
+    removeBtn[removeBtn.length - 1].addEventListener('click', Book.remove);
   }
-  booksArr = bookData;
-}
-loadBooks();
-
-// Prevent Form from Submit
-bookForm.addEventListener('submit', (event) => {
-  event.preventDefault();
-});
 
-// Add a Book
-function AddBook(
-  id = booksArr.length,
-  title = titleBookInput,
-  author = authorBookInput,
-) {
-  createBook(id, title.value, author.value);
+  // Add a Book
+  Add() {
+    const newBook = new Book(this.id, this.title, this.author);
+    newBook.create();
+    booksArr.push(newBook);
+    localStorage.setItem('Books', JSON.stringify(booksArr));
+  }
 
-  booksArr.push({ id, title: title.value, author: author.value });
-  localStorage.setItem('Books', JSON.stringify(booksArr));
-}
+  // Clear Fields
+  static clearField() {
+    titleBookInput.value = '';
+    authorBookInput.value = '';
+  }
 
-// Function to clear Fields
-function clearField() {
-  titleBookInput.value = '';
-  authorBookInput.value = '';
+  static load(bookData = JSON.parse(localStorage.getItem('Books'))) {
+    if (bookData !== null) {
+      bookData.forEach((book) => {
+        const newBook = new Book(book.id, book.title, book.author);
+        newBook.create();
+      });
+    }
+    booksArr = bookData;
+  }
 }
 
 // Event Listeners
 addBtn.addEventListener('click', () => {
   if (titleBookInput.value !== '' && authorBookInput.value !== '') {
-    AddBook();
-    clearField();
+    const newBook = new Book(
+      booksArr.length,
+      titleBookInput.value,
+      authorBookInput.value,
+    );
+    newBook.Add();
+    Book.clearField();
```

Comment:
- [ ] I can't add a book. Please fix the error.

![image](https://user-images.githubusercontent.com/38635158/170129586-af018ece-a3eb-4219-8e7d-992930276645.png)